### ISSUE TITLE
api/v2.0/ps/ Error with <defunct> processes

### ISSUE THREAD

**Issue by Not-Code** [google/cadvisor#2814]
Whenever the GetProcessList calls **ps -e -o user,pid,ppid,stime,pcpu,pmem,rss,vsz,stat,time,comm,psr,cgroup** and a defunct process is present in the returned list, the API fails and the client gets a 500 error with the following output:

`process listing failed: invalid pid "24647": strconv.Atoi: parsing "<defunct>": invalid syntax`

Below an example of the output that is breaking the parsing:

`root     24647 10145 17:24  0.0  0.0     0      0 Z<s  00:00:00 SYNO. <defunct>   3 -`

**Comment by Not-Code** [comment]
This is caused by the following line of code in container.go

```
psr, err := strconv.Atoi(fields[11])
if err != nil {
	return nil, fmt.Errorf("invalid pid %q: %v", fields[1], err)
}
```

and that part has been deleted in master:

#2751 

Closing the issue and waiting for the release.

**Comment by iwankgb** [comment]
@Not-Code I will try to have the fix backported to 0.37 and 0.38.
